src/connectivity/components.c

```diff
@@ -979,6 +979,11 @@ static igraph_error_t igraph_i_decompose_strong(const igraph_t *graph,
  * A vertex is an articulation point if its removal increases
  * the number of (weakly) connected components in the graph.
  *
+ * </para><para>
+ * Use \ref igraph_is_biconnected() if you are only interested in whether the
+ * graph has at least one articulation point: if it is biconnected, it does not
+ * have any articulation points, otherwise it has at least one.
```

Comment:
This is not quite accurate. Once again $K_2$ has no articulation points yet it is biconnected.

Suggestion:

```suggestion
 * Use \ref igraph_is_biconnected() if you are only interested in whether the
 * graph has precisely one biconnected component.
```